src/app/components/shared/form/form.component.ts

```diff
@@ -31,31 +35,68 @@ export class FormComponent extends withUnsubscribe() implements OnInit {
   @Input() public submitLoading: boolean;
```

Comment:
why is this an input?

Reply:
Allows the parent component to disable the submit button while it is loading the response from the submission